src/middleware/validator.js

```diff
@@ -96,15 +96,17 @@ exports.validateInput = validationMap => async (ctx, next) => {
     logger.debug('Successfully validated user input')
   } catch (error) {
     ctx.status = 400
-    ctx.body = error
+    ctx.type = 'json'
+    ctx.body = JSON.stringify({ error: error.message })
     return logger.error(error)
```

Comment:
It might be better to return the actual error message in the logger instead of the error object. This prints out the error message along side the stack trace to where this was throw which isnt useful as this is a valid input error and not a coding error